development/git_lfs.rst

```diff
@@ -0,0 +1,156 @@
+########################################################
+Using Git Large File Storage (LFS) for Data Repositories
+########################################################
+
+DM uses Git LFS to manage test datasets within our :doc:`normal Git workflow </development/workflow>`.
+`Git LFS is developed by GitHub <https://git-lfs.github.com/>`_, though DM uses its own backend storage infrastructure (see `SQR-001: The Git LFS Architecture <http://sqr-001.lsst.io>`_ for background).
+
+All DM repositories should use Git LFS to store binary data, such as FITS files, for :abbr:`CI (Continuous Integration)`.
+Examples of LFS-backed repositories in use are `lsst/afw <https://github.com/lsst/afw>`_, `lsst/hsc_ci <https://github.com/lsst/ci_hsc>`_ and `lsst/testdata_cfht <https://github.com/lsst/testdata_cfht>`_.
+
+This page describes how to use Git LFS for DM development.
+
```

Comment:
It may make sense to list git v1.8.2+ as the one requirement for git-lfs.

I'd generally not add something like this that is already available on their site. But there are clearly systems and users that actively use older versions of git in our organization.